runbot_travis2docker/models/runbot_branch.py

```diff
@@ -0,0 +1,168 @@
+# © 2015 Vauxoo
+#   Coded by: moylop260@vauxoo.com
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+import re
+import os
+
+import requests
+import subprocess
+
+from odoo import fields, models, api, tools
+
+
+class RunbotBranch(models.Model):
+    _inherit = "runbot.branch"
+
+    uses_weblate = fields.Boolean(help='Synchronize with Weblate')
```

Comment:
I'm using our standard of [fields](https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#fields)

`If the technical name of the field (the variable name) is the same to the string of the label, don't put string parameter for new API fields, because it's automatically taken. If your variable name contains "_" in the name, they are converted to spaces when creating the automatic string and each word is capitalized...`